src/app/Common/Services/Interfaces/ITokenService/ITokenService.cs

```diff
@@ -3,12 +3,13 @@
 using Common.DAL.Documents.Token;
 using Common.DAL.Repositories;
 
-namespace Common.Services.Interfaces
+namespace Common.Services.Interfaces.ITokenService
```

Comment:
I think, it's a bit extra to create a separate folder per interface (at least for now). I'd keep models in shared folder, like so:
```
Interfaces
    Models
    ITokenService.cs
    IDocumentService.cs
    ...
```